libs/nillion-chain/client/src/client.rs

```diff
@@ -438,3 +440,44 @@ impl NillionChainClient {
         client.abci_query(path, data, None, false).await.map_err(Into::into)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::client::NillionChainClient;
+
+    #[test]
+    fn test_gas_price_with_exact_multiple() {
+        let price = NillionChainClient::calculate_adjusted_gas_price(4000, 0.025, 0);
+        assert_eq!(price, 100, "Exact multiple should not round up");
+
+        let price = NillionChainClient::calculate_adjusted_gas_price(4000, 0.025, 1);
+        assert_eq!(price, 101, "Exact multiple should not round up");
```

Comment:
This string is outdated. I'd suggest using `rstest` and condensing all of these tests into a single test with proper names in the `case::` macro, that should provide enough info on failure.